src/welcome/WelcomePage.js

```diff
@@ -30,7 +30,7 @@ const WelcomePage = ({ t, doUpdateIpfsApiAddress, apiUrl, ipfsInitFailed, ipfsCo
             </Trans>
             <ApiAddressForm
               t={t}
-              defaultValue={ipfsApiAddress}
+              defaultValue={ipfsApiAddress || ''}
```

Comment:
You should probably do
```
WelcomePage.defaultProps = {
   ipfsApiAddress: ''
}
```

Just to become more bulletproof of future uses (and also to avoid the `||` logic on each re-render, since defaultProps is only handled in the first render.